the-simple-workflow.md

```diff
@@ -0,0 +1,61 @@
+# The S.I.M.P.L.E Workflow
+
+The S.I.M.P.L.E workfow provides a guide for how we should be working with clients at Leaf. It’s not design or development specific, but instead focuses on how we work through identifying problems, solutions to those problems, and then delivering those solutions.
+
+The workflow has a few key themes: a **linkable point of reference**, **client communication**, and **seeking help**.
+
+To help it stick, we’ve ~~forced~~ developed a clever acronym for the workflow - S.I.M.P.L.E!
+
+## Specify the problem
+This might be given to you clearly by the client, but they might need you to **communicate with them** to help get a shared clear definition of the problem.
+
+When you feel you have that shared understanding, ensure it’s clearly defined somewhere linkable. This might be a Basecamp message or todo, or a Trello card, or a Pivotal ticket. The important thing is to provide a **linkable point of reference** where someone can check the status of the work.
```

Comment:
**Suggestion:** This might be a Basecamp message, a Trello card, or a Pivotal ticket.